signer/local/local.go

```diff
@@ -250,7 +253,19 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
-	return s.sign(&safeTemplate, profile)
+	cert, err = s.sign(&safeTemplate, profile)
+
+	if profile.CTLog != "" {
+		var derCert, _ = pem.Decode(cert)
+		var ctclient = client.New(profile.CTLog)
+		chain := []ct.ASN1Cert{derCert.Bytes, s.ca.Raw}
+		var _, err = ctclient.AddChain(chain)
+		if err != nil {
+			return nil, err
```

Comment:
Does the server return specific status codes? Maybe a retry is in order in some cases.